elements/pfe-card/src/pfe-card.scss

```
@@ -162,10 +163,18 @@ $size-small: (
   &__header {
     z-index: 2;
     // Declare the header background color
-    background-color: #{pfe-local(BackgroundColor, $region: header)};
-    color:            #{pfe-local(Color, $region: header)};
+    background-color: pfe-local(BackgroundColor, $region: header);
+    color:            pfe-local($cssvar: Color, $region: header);
+    :host([on="dark"]) & {
+        // Declare the header background color
+      background-color: #{pfe-local(
```

```suggestion
      background-color: pfe-local(BackgroundColor--dark, $region: header);
```

This might be easier to read and looks more like line 166